R/tags.R

```diff
@@ -54,6 +54,19 @@ registerMethods <- function(methods) {
     c("knitr", "knit_print", "shiny.tag"),
     c("knitr", "knit_print", "shiny.tag.list")
   ))
+
+  # TODO: After rlang > 0.4.11 hits CRAN, remove this and replace
+  # with ` #' @importFrom rlang obj_address`
+  # (lionel says rlang:::sexp_address() will be available for the next few years)
+  assign_obj_address()
+  setHook(
+    packageEvent("rlang", "onLoad"),
+    function(...) assign_obj_address()
+  )
+}
+
+assign_obj_address <- function() {
+  assign("obj_address", getFromNamespace("sexp_address", "rlang"), environment(.onLoad))
 }
```

Comment:
After removing the `setHook()`, may as well move this into `.onLoad`, since it only needs to be  called once.